scripts/grounding/.travis.yml.old

```diff
@@ -0,0 +1,16 @@
+sudo: required
```

Comment:
Since we are using Drone for CI, can we remove this file?

Reply:
Yes